mergify_engine/rules.py

```diff
@@ -50,13 +50,13 @@
         voluptuous.Required('users', default=[]): [str],
     }),
     'enforce_admins': voluptuous.Any(None, bool),
-}
+})
 
 # TODO(sileht): We can add some otherthing like
 # automatic backport tag
 # option to disable mergify on a particular PR
 Rule = {
-    'protection': Protection,
+    'protection': ProtectionSchema,
```

Comment:
What I said for Rule is also valid for anything inside the schema. This make Protection attributes no more required.